visualization/app/codeCharta/ui/sortingButton/sortingButton.component.html

```diff
@@ -1,3 +1,7 @@
-<md-button class="box-rounded" ng-click="$ctrl.onButtonClick()">
+<md-button
+	class="box-rounded"
+	ng-click="$ctrl.onButtonClick()"
+	ng-attr-title="{{ $ctrl._viewModel.orderAscending ? 'Ascending' : 'Descending' }}"
```

Comment:
Do you still want to change this to include the toggle part?

Reply:
Oh yes, my bad.

@BridgeAR Feel free to merge now